cloud/disk_manager/test/acceptance/test_runner/eternal_acceptance_test_runner.py

```diff
@@ -33,9 +32,8 @@ def __init__(self, ycp: YcpWrapper, args: argparse.Namespace):
         super(EternalTestCleaner, self).__init__(ycp, args)
         test_type = args.test_type
         disk_name_pattern = re.compile(
-            fr'^acceptance-test-{test_type}-'
```

Comment:
You do not need to remove pattern for old disks then. Otherwise you are introducing breaking changes.